internal/service/comprehend/flex.go

```diff
@@ -0,0 +1,35 @@
+package comprehend
```

Comment:
Trying to understand why `internal/flex` isn't used instead of package-local versions.

Reply:
The package-local versions were written before the `internal/flex` equivalents existed. I'll update